index.js

```diff
@@ -52,15 +52,15 @@ controller.hears('invite', 'direct_mention', (bot, message) => {
 /**
  * Private Onboarding
  */
-controller.on('team_join', onboard);
+controller.on('team_join', () => setTimeout(onboard, 5000));
```

Comment:
not sure this works this way, how are `bot`, `message` vars being passed to onboarding?

Perhaps we should add the delay in the actual onboarding script